canopeum_frontend/src/components/analytics/batch-modal/BatchForm.tsx

```diff
@@ -283,11 +283,42 @@ const BatchForm = ({ handleBatchChange, initialBatch }: Props) => {
         />
       </div>
 
-      <div>
+      <div className='d-flex flex-column'>
         <label className='form-label' htmlFor='batch-image'>
           {t('analyticsSite.batch-modal.images-label')}
         </label>
-        <ImageUpload id='batch-image-upload' imageUrl={batchImageURL} onChange={onImageUpload} />
+        <input
+          id='batch-image'
+          multiple
+          onChange={event => {
+            const newFiles = [...event.target.files ?? []]
+            const allFiles = [...batch.images, ...newFiles]
+
+            setBatch(value => ({
+              ...value,
+              images: allFiles,
+            }))
+          }}
+          type='file'
+        />
+        {batch.images.length > 0 && (
+          <AssetGrid
+            isEditable={{
+              removeFile: (id: number) => {
+                const updatedImages = batch.images
+                  .filter((_, index_: number) => index_ !== id)
```

Comment:
This value being a number is automatically filled in

```suggestion
              removeFile: id => {
                const updatedImages = batch.images.filter((_, index_) => index_ !== id)
```